crates/pop-cli/src/common/wallet.rs

```diff
@@ -46,6 +46,7 @@ mod tests {
 	// This is a helper test for an actual running pop CLI.
 	// It can serve as the "frontend" to query the payload, sign it
 	// and submit back to the CLI.
+	#[ignore]
 	#[tokio::test]
 	async fn sign_call_data() -> anyhow::Result<()> {
```

Comment:
can be removed if its purpose is properly represented elsewhere in the tests.

Reply:
Makes sense. Even more now that we have the front end part too.

Reply:
Removed in [f9851d1](https://github.com/r0gue-io/pop-cli/pull/374/commits/f9851d1ec0bd761e83f0f4ab833dec344b9b5ba2)